src/vignette/http/legacy/route_helpers.clj

```diff
@@ -21,6 +23,22 @@
 (defn archive? [map]
   (= (:zone map) (str "/" archive-dir)))
 
+(defn browser-supports-webp? [request]
```

Comment:
Since these methods basically duplicate the ones in the normal `route_helpers` file, what do you think of moving them to a shared helper file and reusing them in both?